src/ui/zabapgit_css_common.w3mi.data.css

```diff
@@ -674,6 +674,50 @@ div.info-panel .version-marker {
 div.info-panel .update { border:  1px solid; }
 div.info-panel div.info-list td { padding-right: 1em }
 
+/* Error message Panel */
+
+div.message-panel {
+  z-index: 99;
+  box-shadow: 2px 2px 4px 0px hsla(0, 0%, 0%, .1);
+  padding: 12px;
+  margin-left: -48%;
+  position: fixed;
+  bottom: 12px;
+  width: 94%;
+  left: 50%;
+
+  border: 1px solid;
+  border-radius: 5px;
+  border-color: hsl(0, 42%, 64%);
+  background-color: hsla(0, 42%, 90%, 1);
+}
+
+.message-panel .close-btn{
+  position: absolute;
+  right: 20px;
+  bottom: 10px;
+}
```

Comment:
propose to remove `.message-panel .close-btn` style - `position absolute` makes it unrelated to the parent div. Guess the whole design might need some padding tuning at the end, but position absolute is too heavy :)